shared/featureconfig/flags.go

```diff
@@ -150,6 +150,10 @@ var (
 		Name:  "wait-for-synced",
 		Usage: "Uses WaitForSynced for validator startup, to ensure a validator is able to communicate with the beacon node as quick as possible",
 	}
+	enableDebugRPCEndpoints = &cli.BoolFlag{
+		Name:  "enable-debug-rpc-endpoints",
```

Comment:
Is a feature flag appropriate or should you make this a regular flag? 

Feature flags are temporary while you work on the feature